indico/cli/i18n.py

```diff
@@ -335,6 +335,68 @@ def remove_empty_pot_files(path: Path, python=False, javascript=False, react=Fal
         _get_messages_react_pot(path).unlink()
 
 
+# Merge after generating all the .pot files
+def merge_pot_files(output_file, *input_files):
+    """Merge multiple POT files into a single POT file."""
+    merged_catalog = None
+
+    print('\n🗨  Merging POT files...\n')
+
+    for input_file in input_files:
+        with input_file.open('rb') as f:
+            catalog = read_po(f)
+
+            if merged_catalog is None:
+                merged_catalog = catalog
+            else:
+                for message in catalog:
+                    if message.id not in merged_catalog:
+                        merged_catalog[message.id] = message
```

Comment:
I think we don't need this check. The catalog will do the merging for us and as a bonus it also aggregates file locations across the different resources.
```suggestion
                    merged_catalog[message.id] = message
```

Reply:
Indeed, I was also about to comment that we're not adding location information during the merge, and this fixes it.